tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+----
+
+*Bibliographic entries*
+
+In standard documents typically there are two types of references, namely the “normative references” and the “bibliography” (informative references).
+Every bibliographic section must be preceded by the style attribute `[bibliography]` so that bibliographic references are recognized as such. 
+
+A reference entry is usually part of an unordered list (`*`). The bibliographic entry is enclosed in triple square brackets (`[[[]]]`) and contains:
+
+. The anchor name used to reference this entry
+. A document identifier
+
+After the reference, you include the reference text in italics, for example the title of the document.
+
+[source, AsciiDoc]
+----
+* [[[anchor,document identifier or reference tag]]], _reference list text_
+----
+
+.Example for a bibliography section
+[source, AsciiDoc]
+----
+[bibliography]
+== Normative references
+
+* [[[ISO20483,ISO 20483:2013]]], _Cereals and cereal products -- Determination of moisture content -- Reference method_
+* [[[ISO6540,ISO 6540:1980]]]. _Maize -- Determination of moisture content (on milled grains and on whole grains)_
+----
+Gets rendered as:
+* ISO 20483:2013. _Cereals and cereal products — Determination of moisture content — Reference method_
+* ISO 6540:1980. _Maize — Determination of moisture content (on milled grains and on whole grains)_
+
+To cite an entry from your bibliography you type the anchor name like this `<<ISO20483>>`.
+If you want to specify a location within the cited document, you can add https://www.metanorma.org/author/topics/document-format/bibliography/#localities[localities] in the brackets like so: `<<ISO20483, part=IV,chapter=3,paragraph=12>>`.
+
+[[exercise-2-11]]
+include::exercises.adoc[tag=exercise-2-11]
+
+Let’s summarize what we’ve learnt so far.
+>> Start next lesson
+
+=== Summary
+
+We’ve covered a lot of ground, so here is a quick summary for you:
+
+* Each Metanorma document contains metadata in the preamble using the colon notation `:attribute:`.
+* A Metanorma document consists of several predefined sections. The document model of your organization dictates which are optional and which are mandatory. Sections are invoked using equal signs `=`.
+* Blocks are entities that belong together, such as paragraphs, lists, tables, etc. Paragraphs and lists do not need to be marked by delimiters. The remaining block types begin and end with a delimiter, for example four dashes `----`.
+* Inline markup is used for text formatting and references. You can reference:
+** an external source (`URL[Link text]`) 
+** Metadata (`{attribute}`)
+** Places in the document by setting an anchor (`[[anchor-id]]`) and referencing the anchor (`<<anchor-id>>`)
+** Bibliographic entries + 
+[source, AsciiDoc]
+----
+* [[[anchor,document identifier or reference tag]]], _reference list text_
+<<anchor>>
+----
+
+You did a great job so far! Let’s talk about reviewing documents in the next lesson. 
+>> Start next lesson
+
+
+== Reviewing Metanorma Documents
```

Comment:
I am worried about this, because the associated code is immature, and we have never fixed the HTML rendering of comments in 3 years to allow interactive comments. @ronaldtse's call on whether this is included in the tutorial, but I think we shouldn't.